clients/java/src/main/java/io/camunda/zeebe/client/ZeebeClientBuilder.java

```diff
@@ -118,6 +118,8 @@ ZeebeClientBuilder applyEnvironmentVariableOverrides(
    */
   ZeebeClientBuilder overrideAuthority(String authority);
 
+  ZeebeClientBuilder maxMessageSize(int maxSize);
```

Comment:
❌ We'll need to add documentation to clarify to users what happens when they set it and what the default is. It should also be clear what unit is expected.

Reply:
fixed